packages/terra-form-select/src/shared/_Option.jsx

```diff
@@ -105,21 +112,20 @@ const Option = ({
 
   let role = 'option'; // Used for JAWs and VoiceOver on iOS
 
-  /**
-   * VoiceOver in Safari on desktop has issues with role="option" with the combination of the
-   * aria-live section we have and will stutter when reading options.
-   * Switching to role="radio" and role="checkbox" mitigates this behavior.
-   */
-  if (SharedUtil.isSafari() && !('ontouchstart' in window)) {
-    role = 'radio';
+  let itemLabel = display;
+  if (index === 1 || isSelected) {
+    itemLabel = `${expandedStateText} ${display} `;
+    if (!isSelected && SharedUtil.isMac() && SharedUtil.isSafari() && variant !== 'tag' && variant !== 'multiple') {
+      itemLabel = intl.formatMessage({ id: 'Terra.form.select.notselected' }, { text: itemLabel });
+    }
```

Comment:
```suggestion
    if (!isSelected && SharedUtil.isMac() && SharedUtil.isSafari() && variant !== 'tag' && variant !== 'multiple') {
      itemLabel = intl.formatMessage({ id: 'Terra.form.select.notselected' }, { text: itemLabel });
    }
```
move this if block outside of the above if block. so that translation will be available for all the items instead of just being available for 1st item

Reply:
Updated